docs/content/en/blog/news/november-2021-update.md

```diff
@@ -0,0 +1,178 @@
+
+But to provide more flexibility to the developers, especially those are still not able to apply container technology into their development process. Starting from `v0.20.0` PipeCD introduces two new ways to deploy an AWS Lambda function: deploying directly from source code stored in Git, deploying a Zip packing code stored in AWS S3.
+
+{{< tabpane >}}
+{{< tab lang="yaml" header="Using source code stored in a Git repository" >}}
+# Deploy Lambda function directly from source code stored in a Git repository.
+apiVersion: pipecd.dev/v1beta1
+kind: LambdaFunction
+spec:
+  name: SimpleFunction
+  source:
+    git: git@github.com:username/lambda-function-code.git
+    ref: v1.0.0
+    path: hello-world
+{{< /tab >}}
+{{< tab lang="yaml" header="Using Zip of source code stored in S3" >}}
+# Deploy Lambda function by a Zip packing source code stored in S3.
+apiVersion: pipecd.dev/v1beta1
+kind: LambdaFunction
+spec:
+  name: SimpleFunction
+  s3Bucket: pipecd-sample-lambda
+  s3Key: pipecd-sample-src
+  s3ObjectVersion: 1pTK9_v0Kd7I8Sk4n6abzCL
+{{< /tab >}}
+{{< tab lang="yaml" header="Using container image" >}}
+# Deploy Lambda function by using a container image.
+apiVersion: pipecd.dev/v1beta1
+kind: LambdaFunction
+spec:
+  name: SimpleFunction
+  image: ecr.ap-northeast-1.amazonaws.com/lambda-test:v0.0.1
+{{< /tab >}}
+{{< /tabpane >}}
+
+Check out its [documentation](/docs/user-guide/configuring-deployment/lambda) for more details.
+
+##### Automated Deployment Analysis (ADA)
+
+ADA is a good way to automate the process of evaluating the impact of a deployment during its lifetime to help building a robust release process. Any application can be configured to analyze its deployments based on its metrics data, log messages, or results from directly emitting requests to the application.
+
+Until now, PipeCD has been providing a way to allow the developer to configure a list `query` and `threshold` to analyze metrics data of a running deployment. Any query that returns a result exceeding the given threshold will cause the deployment to be failed, and a rollback stage will be executed if needed. Although this way is relatively straightforward, there is a challenge you can see here; it isn't easy to find an appropriate threshold value to use. And software is constantly changing time over time that making those values could be changed dynamically. Therefore, some new analysis strategies `PREVIOUS`, `CANARY_BASELINE`, `CANARY_PRIMARY` have been introduced since `v0.20.0`. They allow analyzing without requiring any threshold values.
```

Comment:
```suggestion
Until now, PipeCD has been providing a way to allow the developer to configure a list of `query` and `threshold` to analyze metrics data of a running deployment. Any query that returns a result exceeding the given threshold will cause the deployment to be failed, and a rollback stage will be executed if needed. Although this way is relatively straightforward, there is a challenge you can see here; it isn't easy to find an appropriate threshold value to use since software is constantly changing time over time that making those values could be changed dynamically. Therefore, some new analysis strategies `PREVIOUS`, `CANARY_BASELINE`, `CANARY_PRIMARY` have been introduced since `v0.20.0`. They allow analyzing without requiring any threshold values.
```
nits